src/renderer/components/Message.js

```diff
@@ -147,9 +147,20 @@ function convertMessageStatus (s) {
   }
 }
 
-function convertContentType (filemime) {
+function convertContentType (message) {
+  var filemime = message.filemime
   if (!filemime) return 'image/jpg'
-  if (filemime === 'application/octet-stream') return 'audio/ogg'
+  if (filemime === 'application/octet-stream') {
+    var s = message.msg.viewType
+    switch (s) {
```

Comment:
Only a very minor thing, why doing the reference to `s`? I would just do an `switch (message.msg.viewType), i think it's clean enough.